### PR TITLE
Added op to track connection handling

### PR DESCRIPTION
**Pull request by oxtoacart**
This will give us a better idea of what's going on with proxies.

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -145,17 +146,30 @@ func (s *Server) handle(conn net.Conn) {
...
 
 func (s *Server) doHandle(conn net.Conn, isWrapConn bool, wrapConn listeners.WrapConn) {
+	log.Debug(conn)
+	log.Debug(conn.RemoteAddr())
+	clientIP := ""
+	remoteAddr := conn.RemoteAddr()
+	if remoteAddr != nil {
+		clientIP, _, _ = net.SplitHostPort(remoteAddr.String())
+	}
+	op := ops.Begin("http_proxy_handle").Set("client_ip", clientIP)
```
Comment (myleshorton) [inline server/server.go]: Do we maybe only want to trigger ops in cases where we definitely get a client IP?

Comment (oxtoacart) [reply server/server.go]: The only case where we won't get a client IP is during unit testing where we sometimes use mock connections. In production, there should always be a client IP.

File: server/server.go [hunk 2]
```diff
@@ -145,17 +146,30 @@ func (s *Server) handle(conn net.Conn) {
 }
 
 func (s *Server) doHandle(conn net.Conn, isWrapConn bool, wrapConn listeners.WrapConn) {
+	log.Debug(conn)
+	log.Debug(conn.RemoteAddr())
```
Comment (joesis) [inline server/server.go]: Unnecessary logging

**Comment by joesis** [comment]
LGTM other than two unnecessary logging.

**Pull request merged**